### ISSUE TITLE
more than one slash inserts "null"

### ISSUE THREAD

**Issue by AurielleP** [spencermountain/compromise#745]
```
const doc = nlp(Ramazanoğlu Mah. Mahsus Sk. No:1 Pendik / İSTANBUL / TÜRKİYE);
doc.debug();

// results in:

"No:1 Pendik / İSTANBUL null/ TÜRKİYE"
'No:1'    -  Noun, Singular
'Pendik / İSTANBUL'  -  ProperNoun, Noun, Singular
'null/ TÜRKİYE'  -  Noun, Singular

```

**Comment by spencermountain** [comment]
are `Sk` and `Mah` i18n abbreviations?

**Comment by AurielleP** [comment]
no - they are just street address part abbrev
```
mah. - mahallesi (district)
mh. - mahallesi (district)
blv. - bulvarı (boulevard)
cad. - caddesi (road)
cd. - caddesi (road)
sk. - sokak (alley)
ap. - apartmanı (apartment)
kat - floor. Kat 1 is 2nd floor by American method
```

they were tagged fine - i just excluded them to reduce the example output to only the part that was a bug (the slash parsing)

**Comment by spencermountain** [comment]
hey, this appears fixed now, in 13.2.0
```js
const doc = nlp(`Ramazanoğlu Mah. Mahsus Sk. No:1 Pendik / İSTANBUL / TÜRKİYE`)
console.log(doc.terms().out('array'))
[ 'Ramazanoğlu',
  'Mah.',
  'Mahsus',
  'Sk.',
  'No:1',
  'Pendik',
  '/',
  'İSTANBUL',
  '/',
  'TÜRKİYE' ]
```
maybe we should try to join lonsesome slashes in the future, but for now I think that's the desired behaviour
let me know if i'm wrong
cheers
